server/src/services/notification.services.js

```diff
@@ -68,6 +68,71 @@ export const sendNotificationApplicationDecision = async (applicationId, status)
   }
 };
 
+export const sendNotificationApplicationDecisionSupervisor = async (applicationId, status) => {
+  try {
+    const subject = "Thesis Management - Supervisor application decision";
+    const title = await getProposalTitleByApplicationId(applicationId);
+    const htmlMessage = `
+    <html>
+      <head>
+        <style>
+          body {
+            font-family: Arial, sans-serif;
+            background-color: #f4f4f4;
+            color: #333;
+          }
+          .container {
+            width: 80%;
+            heigth: 100vh;
+            margin: auto;
+            padding: 2%;
+            background-color: #fff;
+            border-radius: 5px;
+            box-shadow: 0 2px 5px rgba(0,0,0,0.1);
+          }
+          h1 {
+            color: #007bff;
+          }
+        </style>
+      </head>
+      <body>
+        <div class="container">
+          <h1>Application of proposal ${title} Status Update</h1>
+          <p>
+            Application of proposal ${title} has been updated to "${status}". As a Co-supervisor you can now manage it.
+          </p>
+        </div>
+      </body>
+    </html>
```

Comment:
The HTML codes of the email body in this part are duplicated to a large extent with a 'sendNotificationApplicationDecision' function and can be converted into a function and avoid about 40 lines of duplicate code.